Mayhemfile

```diff
@@ -0,0 +1,12 @@
+image: ghcr.io/rishabh1024/drlibs-fuzz:latest
```

Comment:
You may omit the image tag as this will be populated during runtime.

Reply:
I have made the changes